zkvm/src/types.rs

```diff
@@ -146,6 +148,13 @@ impl ScalarWitness {
         }
     }
 
+    pub fn to_scalar(self) -> Scalar {
+        match self {
+            ScalarWitness::Integer(i) => <SignedInteger as Into<Scalar>>::into(i),
```

Comment:
`i.into()` should be unambiguous, i think, since compiler sees that the intended result is Scalar.